A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js

```diff
@@ -296,9 +283,22 @@ class A320_Neo_EICAS extends Airliners.BaseEICAS {
         // switch page when desired page was changed, or new Failure detected
         if ((this.pageNameWhenUnselected != prevPage && this.currentPage == -1) || (this.PrevFailPage !== sFailPage)) {
             this.SwitchToPageName(this.LOWER_SCREEN_GROUP_NAME, this.pageNameWhenUnselected);
+        }
 
+        // ECAM all button
+        this.ecamAllButtonState = SimVar.GetSimVarValue("L:A32NX_ECAM_ALL_Push_IsDown", "Bool");
+
+        if (this.ecamAllButtonState && !this.ecamAllButtonPrevState) { // button press
+            this.ecamCycleInterval = setInterval(() => {
+                this.changePage(this.lowerScreenPages[(this.currentPage + 1) % this.lowerScreenPages.length].name);
+            }, 1000);
+            SimVar.SetSimVarValue("L:A32NX_ECAM_CYCLE", "Bool", true);
+        } else if (!this.ecamAllButtonState && this.ecamAllButtonPrevState) { // button release
+            clearInterval(this.ecamCycleInterval);
+            SimVar.SetSimVarValue("L:A32NX_ECAM_CYCLE", "Bool", false);
```

Comment:
`L:A32NX_ECAM_CYCLE`

Do you plan on using this SimVar for something later on, or is it required for something else?
If not required then removing it is a good idea.

Reply:
Not required, had it for debugging and thought it might come in handy later. Want me to remove it?